ccd-definition/CaseField/CareSupervision/caseField.json

```diff
@@ -410,15 +456,17 @@
     "Label": "Gatekeeper",
     "FieldType": "Collection",
     "FieldTypeParameter": "GatekeeperEmail",
-    "SecurityClassification": "Public"
+    "SecurityClassification": "Public",
+    "Searchable": "N"
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "ID": "dateAndTimeSubmitted",
     "Label": "Date submitted",
     "FieldType": "DateTime",
-    "SecurityClassification": "Public"
+    "SecurityClassification": "Public",
+    "Searchable": "N"
```

Comment:
it should be searchable. XUI do an ordering based on this fields, and this is on server side, so all fileds in results needs to be sortable